projects/demo/src/modules/components/app-bar/examples/5/index.html

```diff
@@ -0,0 +1,84 @@
+<tui-segmented [(activeItemIndex)]="activeItemIndex">
+    <button type="button">IOS</button>
+    <button type="button">Android</button>
+</tui-segmented>
+
+<div
+    tuiDynamicHeaderContainer
```

Comment:
Can this artifact be fixed ?
<img width="342" alt="example" src="https://github.com/user-attachments/assets/0d4d8743-bb69-4912-aeea-cf96e5443f46" />

https://github.com/user-attachments/assets/ed61ffee-f890-4d31-b64b-fa9aea9a6b40